chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict, List
+
+from chaoslib.types import Configuration, Secrets
+from googleapiclient import discovery
+
+from chaosgcp import load_credentials
+
+__all__ = ["update_service"]
+
+
+def update_service(
+    secrets: Secrets = None,
+    project: str = None,
```

Comment:
that should be coming from the configuration

Reply:
see https://github.com/chaostoolkit-incubator/chaostoolkit-google-cloud-platform/blob/master/chaosgcp/artifact/probes.py#L29